apps/docs/docusaurus.config.ts

```diff
@@ -110,6 +101,14 @@ const config: Config = {
     ],
   ],
   themeConfig: {
+    matomo: {
+      matomoUrl: 'https://webstats.uzh.ch',
+      siteUrl: 'https://www.klicker.uzh.ch',
+      siteId: '356',
+      phpLoader: 'matomo.php',
+      jsLoader: 'matomo.js',
+      dev: true,
```

Comment:
Is the coderabbit comment regarding the use of the dev-mode correct here or is this supposed to work like this?